CatalogService/BLL/ItemService.cs

```diff
@@ -31,6 +33,7 @@ public async Task<ItemEntity> Update(int id, ItemUpdateDto update)
         if (!await _repository.Exists(id)) { throw new BadRequestException($"Invalid item ID: {id}"); }
         await ValidateItem(update);
         await _repository.Update(id, update);
+        _messagingService.Send(new ItemUpdatedMessage { ItemId = id, Update = update });
```

Comment:
What would happen if the application would stop executing at this moment of time? So, we successfully updated the database, but after this, the code stopped working.